### PR TITLE
Update LaravelLocalization.php

### PR DESCRIPTION
**Pull request by lotestudio**
Just found that .utf8 does not work on my servers with lowercase.
I think that setlocale second argument must be in format de_DE.UTF-8

### CODE REVIEW

File: src/Mcamara/LaravelLocalization/LaravelLocalization.php [hunk 1]
```diff
@@ -167,7 +167,8 @@ public function setLocale( $locale = null )
         // Regional locale such as de_DE, so formatLocalized works in Carbon
         $regional = $this->getCurrentLocaleRegional();
         if($regional)
-            setlocale(LC_TIME, $regional.'.utf8');
+            setlocale(LC_TIME, $regional.'.UTF-8');
+            setlocale(LC_MONETARY, $regional.'.UTF-8');
```
Comment (fisharebest) [inline src/Mcamara/LaravelLocalization/LaravelLocalization.php]: Does the `if` need `{` and `}`

**Comment by xitude** [comment]
I agree with @fisharebest, @lotestudio do you need to wrap your if statement.

File: src/Mcamara/LaravelLocalization/LaravelLocalization.php (hunk 1 above)
Comment (lotestudio) [inline src/Mcamara/LaravelLocalization/LaravelLocalization.php]: Uf, fixed! Thanks!

**Comment by mcamara** [comment]
Thanks for your help! Merged!

**Pull request merged**